### ISSUE TITLE
A few questions in your paper and code

### ISSUE THREAD

**Issue by xyz189411yt** [SAI990323/TALLRec#14]
1. You have mentioned in your paper that the learning rate is 1e-3 but which in instruct_7B.sh is 1e-4, which is exactly the true parameter you used?
2. The optimizer funciton in the paper is Adam but AdamW in training file.
3. The loss function in the paper is MSE, but as far as I know, most LM using CrossEntropy loss, and I believe I didn't find where you defined the loss function, but instead found the model's predefined CrossEntropy loss function inside the LLaMA.

I don't know if I didn't find the right file or if your code or paper needs to be updated

**Comment by SAI990323** [comment]
Thanks for your reminder. There is a typo in the arXiv preprint version. We applied this setting to all baseline methods (traditional recommendation models). For the LLM-based experiments, the hyper-parameters are available on GitHub. We will modify these typo errors and add the LLaMA hyper-parameters to the paper in our camera-ready version.
